packages/web-client/app/components/card-pay/deposit-workflow/transaction-amount/index.ts

```diff
@@ -124,7 +124,7 @@ class CardPayDepositWorkflowTransactionAmountComponent extends Component<Workflo
 
   validate() {
     this.validationMessage = validateTokenInput(this.amount, {
-      min: new BN(0),
+      tokenSymbol: this.currentTokenSymbol,
       max: this.currentTokenBalance,
     });
```

Comment:
Maybe this should also use the `balance` validation option? For a more consistent validation message.

Reply:
Agreed, done in bf2428e, thanks